ci/ci-ext-custom.yml

```
@@ -10,8 +10,30 @@
 include:
   - remote: 'https://gitlab.com/cscs-ci/recipes/-/raw/master/templates/v2/.ci-ext.yml'
 
-# Need custom definition as remote adds SLURM_MPI_TYPE: pmi2
+.dlaf-container-runner-eiger:
+  extends: .container-runner-eiger
+  variables:
+    SLURM_CONSTRAINT: mc
+
 .dlaf-container-runner-daint-gh200:
   extends: .container-runner-daint-gh200
   variables:
+    CRAY_CUDA_MPS: 1
```

```suggestion
    CSCS_CUDA_MPS: 1
```
`CRAY_CUDA_MPS` still works, but `CSCS_CUDA_MPS` is preferred (https://confluence.cscs.ch/spaces/KB/pages/868813057/Runners#Runners-container-runner).